src/server/game/Miscellaneous/SharedDefines.h

```diff
@@ -4258,60 +4276,344 @@ enum CreatureEliteType
-    HOLIDAY_FIREWORKS_SPECTACULAR    = 62,
-    HOLIDAY_FEAST_OF_WINTER_VEIL     = 141,
-    HOLIDAY_NOBLEGARDEN              = 181,
-    HOLIDAY_CHILDRENS_WEEK           = 201,
-    HOLIDAY_CALL_TO_ARMS_AV          = 283,
-    HOLIDAY_CALL_TO_ARMS_WS          = 284,
-    HOLIDAY_CALL_TO_ARMS_AB          = 285,
-    HOLIDAY_FISHING_EXTRAVAGANZA     = 301,
-    HOLIDAY_HARVEST_FESTIVAL         = 321,
-    HOLIDAY_HALLOWS_END              = 324,
-    HOLIDAY_LUNAR_FESTIVAL           = 327,
-    // HOLIDAY_LOVE_IS_IN_THE_AIR    = 335, unused/duplicated
-    HOLIDAY_FIRE_FESTIVAL            = 341,
-    HOLIDAY_CALL_TO_ARMS_EY          = 353,
-    HOLIDAY_BREWFEST                 = 372,
-    HOLIDAY_DARKMOON_FAIRE_ELWYNN    = 374,
-    HOLIDAY_DARKMOON_FAIRE_THUNDER   = 375,
-    HOLIDAY_DARKMOON_FAIRE_SHATTRATH = 376,
-    HOLIDAY_PIRATES_DAY              = 398,
-    HOLIDAY_CALL_TO_ARMS_SA          = 400,
-    HOLIDAY_PILGRIMS_BOUNTY          = 404,
-    HOLIDAY_WOTLK_LAUNCH             = 406,
-    HOLIDAY_DAY_OF_DEAD              = 409,
-    HOLIDAY_CALL_TO_ARMS_IC          = 420,
-    HOLIDAY_LOVE_IS_IN_THE_AIR       = 423,
-    HOLIDAY_KALU_AK_FISHING_DERBY    = 424,
-    HOLIDAY_CALL_TO_ARMS_BFG         = 435,
-    HOLIDAY_CALL_TO_ARMS_TP          = 436,
-    HOLIDAY_RATED_BG_15_VS_15        = 442,
-    HOLIDAY_RATED_BG_25_VS_25        = 443,
-    HOLIDAY_ANNIVERSARY_7_YEARS      = 467,
-    HOLIDAY_DARKMOON_FAIRE_TEROKKAR  = 479,
-    HOLIDAY_ANNIVERSARY_8_YEARS      = 484,
-    HOLIDAY_CALL_TO_ARMS_SM          = 488,
-    HOLIDAY_CALL_TO_ARMS_TK          = 489,
-    //HOLIDAY_CALL_TO_ARMS_AV        = 490,
-    //HOLIDAY_CALL_TO_ARMS_AB        = 491,
-    //HOLIDAY_CALL_TO_ARMS_EY        = 492,
-    //HOLIDAY_CALL_TO_ARMS_AV        = 493,
-    //HOLIDAY_CALL_TO_ARMS_SM        = 494,
-    //HOLIDAY_CALL_TO_ARMS_SA        = 495,
-    //HOLIDAY_CALL_TO_ARMS_TK        = 496,
-    //HOLIDAY_CALL_TO_ARMS_BFG       = 497,
-    //HOLIDAY_CALL_TO_ARMS_TP        = 498,
-    //HOLIDAY_CALL_TO_ARMS_WS        = 499,
-    HOLIDAY_ANNIVERSARY_9_YEARS      = 509,
-    HOLIDAY_ANNIVERSARY_10_YEARS     = 514,
-    HOLIDAY_CALL_TO_ARMS_DG          = 515,
-    //HOLIDAY_CALL_TO_ARMS_DG        = 516
+    HOLIDAY_NONE                                   = 0,
+
+    HOLIDAY_FIREWORKS_SPECTACULAR                  = 62,
+    HOLIDAY_FEAST_OF_WINTER_VEIL                   = 141,
+    HOLIDAY_NOBLEGARDEN                            = 181,
+    HOLIDAY_CHILDRENS_WEEK                         = 201,
+    HOLIDAY_CALL_TO_ARMS_ALTERAC_VALLEY            = 283,
+    HOLIDAY_CALL_TO_ARMS_WARSONG_GULCH             = 284,
+    HOLIDAY_CALL_TO_ARMS_ARATHI_BASIN              = 285,
+    HOLIDAY_HARVEST_FESTIVAL                       = 321,
+    HOLIDAY_HALLOWS_END                            = 324,
+    HOLIDAY_LUNAR_FESTIVAL                         = 327,
+    // HOLIDAY_LOVE_IS_IN_THE_AIR                     = 335, duplicated
+    HOLIDAY_MINDSUMMER_FIRE_FESTIVAL               = 341,
```

Comment:
typo? - Midsummer Fire Festival should be correct

Reply:
https://wow.tools/dbc/?dbc=holidays&build=9.0.2.37176#page=1&colFilter[0]=341

![sdd](https://user-images.githubusercontent.com/15787347/105877276-cb610680-5fcd-11eb-830d-5956a1828301.png)

Reply:
You have `MINDSUMMER`, not `MIDSUMMER` hahah

Reply:
hahahaha, I did not see it.

Reply:
Funny idea. Thank you. I will keep it on my mind until next summer... 🌞 😄